### PR TITLE
feat(DIST-725): Add popover

### PR DESCRIPTION
**Pull request by mrisoli**
## Description

Adds popover widget to next branch

### CODE REVIEW

**Review by antoniofull — APPROVED** [review]

File: packages/demo-html/public/popover-html.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Static HTML Demo</title>
+    <style>
+      #wrapper {
```
Comment (mathio) [inline packages/demo-html/public/popover-html.html:7]: `#wrapper` is not required for this page nor for `popover-js.html`.

File: packages/embed/src/factories/create-popover/popover-options.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+     *
+     * @type {string}
+     */
+    buttonColor?: string
+    /**
+     * Specify the image url for the popover trigger button icon
+     *
+     * @type {string}
+     */
+    customIcon?: string
```
Comment (mathio) [inline packages/embed/src/factories/create-popover/popover-options.ts:47]: We should also load those options from attributes in [loadOptionsFromAttributes](https://github.com/Typeform/embed/blob/6d7c673b0694318525c5b9774f885c319f29eb10/packages/embed/src/utils/load-options-from-attributes.ts#L58).

Comment (mrisoli) [reply packages/embed/src/factories/create-popover/popover-options.ts:47]: Should they be included if they are popover specific?

Comment (mrisoli) [reply packages/embed/src/factories/create-popover/popover-options.ts:47]: nvm now I understood how those options are loaded

**Pull request merged**